pom.xml

```diff
@@ -53,6 +53,7 @@
         <version.org.jboss.galleon>5.1.0.Final</version.org.jboss.galleon>
         <version.org.wildfly.galleon-plugins>6.4.2.Final</version.org.wildfly.galleon-plugins>

```

Comment:
And maybe these Galleons dependency will need to be updated according to those of the wildfly-maven-plugin and wildfly-jar-maven-plugin? Not sure though, but I think this should be worth checking.